src/Components/Components/src/ComponentBase.cs

```diff
@@ -148,7 +148,7 @@ protected virtual Task OnAfterRenderAsync()
         /// </summary>
         /// <param name="workItem">The work item to execute.</param>
         protected Task Invoke(Action workItem)
```

Comment:
Shouldn't we also rename this to `InvokeAsync`?

Reply:
Yes! this was just missed. Glad someone's watching.